docs/advanced-configuration.md

```diff
@@ -515,6 +515,37 @@ autodetect:
   prometheus: true
 ```
 
+## Using feature gates
+Enable or disable features of the otel-collector agent, clusterReceiver, and gateway (respectively) using feature
+gates. Use the agent.featureGates, clusterReceiver.featureGates, and gateway.featureGates configs to enable or disable
+features, these configs will be used to populate the otelcol binary startup argument "--feature-gates". For more
+details see the
+[feature gate documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/featuregate/README.md).
+
+Helm Install Example:
+```bash
+helm install {name} --set agent.featureGates=feature1,clusterReceiver.featureGates=+feature2,gateway.featureGates=-feature3 {other_flags}
```

Comment:
Hmm.. the feature gates are also separated by commas, so maybe split it to several --set args like

```suggestion
helm install {name} --set agent.featureGates=+feature1,-feature2 --set=clusterReceiver.featureGates=+feature1,-feature2 {other_flags}
```

Reply:
Good point. We do have to escape commas (special character), so added that too.